chart/values.yaml

```diff
@@ -0,0 +1,102 @@
+role: for-test
+replicas: 1
+
+images:
+  etcd: quay.io/coreos/etcd:v3.3.9
+  etcd-backup-restore: eu.gcr.io/gardener-project/gardener/etcdbrctl:0.3.0
+
+backup:
+  schedule: "*/5 * * * *" # cron standard schedule
```

Comment:
Will please comments as per the standard mentioned [here](https://docs.helm.sh/chart_best_practices/#document-values-yaml).